charts/aws-ebs-csi-driver/values.yaml

```diff
@@ -13,40 +13,46 @@ sidecars:
     env: []
     image:
       repository: k8s.gcr.io/sig-storage/csi-provisioner
-      tag: "v2.1.1"
+      tag: v2.2.2
+      pullPolicy: IfNotPresent
     logLevel: 2
     resources: {}
   attacher:
     env: []
     image:
       repository: k8s.gcr.io/sig-storage/csi-attacher
-      tag: "v3.1.0"
+      tag: v3.2.1
+      pullPolicy: IfNotPresent
     logLevel: 2
     resources: {}
   snapshotter:
     env: []
     image:
       repository: k8s.gcr.io/sig-storage/csi-snapshotter
-      tag: "v3.0.3"
+      tag: v3.0.3
+      pullPolicy: IfNotPresent
     logLevel: 2
     resources: {}
   livenessProbe:
     image:
       repository: k8s.gcr.io/sig-storage/livenessprobe
-      tag: "v2.2.0"
+      tag: v2.3.0
+      pullPolicy: IfNotPresent
     resources: {}
   resizer:
     env: []
     image:
       repository: k8s.gcr.io/sig-storage/csi-resizer
-      tag: "v1.0.0"
+      tag: v1.2.0
+      pullPolicy: IfNotPresent
```

Comment:
I think `Always` is wrong value with fixed image version, and taking the risk of Docker Hub API rate limiting

Reply:
I don't believe that `k8s.gcr.io` is docker hub.  I think it is google cloud.  Also, I believe, if there is already a local copy of the image and the digest for the image hasn't changed it won't pull but instead reuse the local cached images since it hasn't changed.

Reply:
Ok if you think it is reasonable to have `csi-resizer` using `Always` as default image pull policy and all other containers using `IfNotPresent` then I will do it